Example/Stanwood_Dialog_iOS/AppDelegate.swift

```diff
@@ -57,11 +53,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
                 .set(okText: accept)
                 .set(faceUrl: faceUrlString)
                 .set(bannerUrl: bannerUrlString)
-                .buildAppStoreUrl(with: appID)
+                .set(appID: "1316369720")
```

Comment:
have you confirmed this isn't an active appID?
probably a good idea to replace the urls with generic google results or something to remove our API instance ids